oe_theme.theme

```diff
@@ -1198,7 +1200,7 @@ function _oe_theme_preprocess_search_input_text(array $element): array {
   $ecl_array = [
     'id' => $element['#attributes']['id'],
     'name' => $element['#attributes']['name'],
-    'disabled' => isset($element['#attributes']['disabled']) ? $element['#attributes']['disabled'] : FALSE,
+    'disabled' => $element['#attributes']['disabled'] ?? FALSE,
```

Comment:
Fixes of phpcs issues:
```
FOUND 3 ERRORS AFFECTING 3 LINES
----------------------------------------------------------------------
 1203 | ERROR | [x] Use null coalesce operator instead of ternary operator.
 1304 | ERROR | [x] Use null coalesce operator instead of ternary operator.
 1328 | ERROR | [x] Use null coalesce operator instead of ternary  operator.
```